spec/syntax.md

```diff
@@ -808,17 +808,20 @@ Examples:
 >```
 > This has a {$variable}
 >```
-> A function:
+>
+> A default function:
 > ```
-> This has a {:function}
+> This has an {42 :integer}
 > ```
-> An add-on function from the `icu` namespace:
+>
+> A function from the `icu` namespace:
 > ```
 > This has a {:icu:function}
```

Comment:
Probably better to use the examples namespace than `icu`, since `icu` doesn't have a `function` function.

```suggestion
> A function from the `ns` namespace:
> ```
> This has a {:ns:function}
```